Pr: localize views - mailers

pavanjoshi914 opened the pull request:
assignment mailer
groups mailer
pending invitation mailer
user mailer
correcting duplication of keys

Fixes #

#### Describe the changes you have made in this PR -

### Screenshots of the changes (If any) -

* Group Mailer -

create group

![image](https://user-images.githubusercontent.com/55848322/127507050-8fe79e39-9bb9-439f-aa80-94d969a11097.png)

add member

![image](https://user-images.githubusercontent.com/55848322/127507608-bbf50965-f63d-4025-abb6-296ab132ed6a.png)

* Assignment mailer

create assignment

![image](https://user-images.githubusercontent.com/55848322/127509072-38ae425e-3419-4129-b9af-18d66210fcaa.png)

update assignment

![image](https://user-images.githubusercontent.com/55848322/127509650-ba932464-1a4e-4a6d-91a4-928cc2ccb237.png)


Note: Please check **Allow edits from maintainers.** if you would like us to assist in the PR.

pavanjoshi914 committed:
Pr: localize views - mailers
assignment mailer
groups mailer
pending invitation mailer
user mailer
correcting duplicaiton of keys

Shivansh2407 commented on config/locales/views/mailer/hi.yml:
```diff
@@ -0,0 +1,39 @@
+  mailer:
+    welcome_heading: "सर्किटवर्स में आपका स्वागत है, %{name}"
+    keep_building: "निर्माण करते रहे"
+    view_project: "परियोजना देखें"
```
I think we should make use of प्रोजेक्ट instead of परियोजना, to make it less complex.

Shivansh2407 replied:
We can replace परियोजना to प्रोजेक्ट everywhere in the file.

pavanjoshi914 replied:
I have used परियोजना from starting in other modules also as suggested by @ShreyaPrasad1209 shall I change this everywhere?

pavanjoshi914 replied:
@ShreyaPrasad1209 and me decided to use परियोजना earlier. so I used this in other modules too, shall I change this everywhere or only for mailers.

if the change is required I will create new pr as change will push different files for other modules too..

Shivansh2407 replied:
@pavanjoshi914 let it be परियोजना for now, because we would have to change it in other files as well.

Shivansh2407 commented on config/locales/views/mailer/hi.yml:
```diff
@@ -0,0 +1,39 @@
+    member_added_to_group_html: "आपको <span class='mailer-highlighted-text'>%{group_name}</span> नाम के एक नए समूह में जोड़ा गया है। <br>
+    समूह का मार्गदर्शन <span class='mailer-highlighted-text'>%{user_name}</span> द्वारा किया जाता है।"
+  assignment_mailer:
+    assignment_created_html: "<span class='mailer-highlighted-text'>%{name} (%{deadline})</span> नाम का एक की नयी असाइनमेंट <span class='mailer-highlighted-text'> %{group_name} में असाइन की की गयी है </span>. <br>
```
Please check the sentence structure again and update this. It has multiple same words like की की and grammatical mistakes.

Shivansh2407 commented on config/locales/views/mailer/hi.yml:
```diff
@@ -0,0 +1,39 @@
+      project_forked_html: "आपने <span class='mailer-highlighted-text'>%{original_author}</span> से एक नयी परियोजना सफलतापूर्वक फोर्क कि है। <br>
+      आपके फोर्क किए गए परियोजना का नाम <span class='mailer-highlighted-text'>%{forked_name}</span> है और परियोजना का एक्सेस टाइप <span class='mailer-highlighted-text'>%{access_type}</span > है।"
+    new_project_email:
+      project_created_html: "आपने सफलतापूर्वक नयी परियोजना बना लि है, <span class='mailer-highlighted-text'>%{name}</span>।"
```
लि -> ली

Shivansh2407 reviewed (changes requested):
Nice work @pavanjoshi914 :+1: I have suggested a few changes here, please update them and we'll be good to go! Thanks :tada:

pavanjoshi914 committed:
Merge remote-tracking branch 'upstream/master' into localize-views-mailer

ShreyaPrasad1209 commented on config/locales/views/mailer/hi.yml:
```diff
@@ -0,0 +1,39 @@
+en:
+  mailer:
+    welcome_heading: "सर्किटवर्स में आपका स्वागत है, %{name}"
```
Can this name be converted to Hindi? It’s showing I’m English even in Hindi translations

ShreyaPrasad1209 commented on config/locales/views/mailer/hi.yml:
```diff
@@ -0,0 +1,39 @@
+    custom_email:
+      hello_user: "नमस्ते, %{name}"
+    featured_circuit_email:
+      circuit_featured_html: "वाह! आपकी परियोजना, <span class='mailer-highlighted-text'>%{name}</span> मंच पर प्रदर्शित कि गयी है।"
```
Here it should be badi e in “ कि” instead of choti e ki matra

ShreyaPrasad1209 commented on config/locales/views/mailer/hi.yml:
```diff
@@ -0,0 +1,39 @@
+    featured_circuit_email:
+      circuit_featured_html: "वाह! आपकी परियोजना, <span class='mailer-highlighted-text'>%{name}</span> मंच पर प्रदर्शित कि गयी है।"
+    forked_project_email:
+      project_forked_html: "आपने <span class='mailer-highlighted-text'>%{original_author}</span> से एक नयी परियोजना सफलतापूर्वक फोर्क कि है। <br>
```
Same issue of “ कि” here also

ShreyaPrasad1209 commented on config/locales/views/mailer/hi.yml:
```diff
@@ -0,0 +1,39 @@
+      project_created_html: "आपने सफलतापूर्वक नयी परियोजना बना लि है, <span class='mailer-highlighted-text'>%{name}</span>।"
+    welcome_email:
+      know_more: "अधिक जानने के लिए"
+      welcome_text_html: "साइन अप करने के लिए धन्यवाद।<br> सर्किटवर्स के साथ आप साधारण गेट से लेकर काम्प्लेक्स सेक्वेंशिअल सर्किट्स, प्लॉट टाइमिंग डायग्राम, आटोमेटिक सर्किट जनरेशन, स्टैण्डर्ड आईसी के बारे में अधिक, और बहुत कुछ जानिए, आप सर्किटवर्स के साथ मुफ्त में डिजिटल लॉजिक सर्किट डिजाइन और बहुत कुछ कर सकते हैं!"
```
After “dhyanwad” maybe add an exclamation mark!

ShreyaPrasad1209 commented on config/locales/views/mailer/hi.yml:
```diff
@@ -0,0 +1,39 @@
+      design_heading: "डिज़ाइन करना"
+      design_description: "क्या आपको पता नहीं कैसे डिजिटल सर्किट डिजाइन करना है? सर्किटवर्स, शुरुआती लोगों के लिए उपयोग करना आसान है। सर्किटवर्स डॉक्यूमेंटेशन आपको जल्दी से शुरू करने में मदद करेगा।"
+      learn_heading: "सीखना"
+      learn_description: "क्या आप डिजिटल सर्किट के बारे में अधिक जानना चाहते हैं? सर्किटवर्स इंटरैक्टिव बुक आपको डिजिटल सर्किट कैसे काम करती है, इसकी पूरी समझ देती है।"
```
Rephrase this sentence to - “Digital circuits kaise kaam karti h, iski poori samjh CircuitVerse interactive book deti hai. “

ShreyaPrasad1209 commented on config/locales/views/mailer/hi.yml:
```diff
@@ -0,0 +1,39 @@
+  pending_invitation_email:
+    main_heading: "सर्किटवर्स में आपका स्वागत है"
+    main_description_html: "आपको %{user_name} द्वारा %{name} नाम के एक नए समूह में जोड़ा गया है।<br><br>
+     डिजिटल सर्किट की दुनिया को एक्स्प्लोर करने के लिए सर्किटवर्स के साथ साइन अप करें। एक बार जब आप सर्किटवर्स के साथ साइन अप कर लेते हैं तो आप समूह का उपयोग कर सकते हैं।"
```
Maybe rephrase this to - digital circuit ki duniya mei khoj karne ke liye” keeping rest of the sentence same

ShreyaPrasad1209 reviewed (changes requested):
@pavanjoshi914 everything looks good to me except for few things. Please have a look and let me know what you think!

pavanjoshi914 committed:
localize views - mailers
localize all mailers
correct position of css so that sidekiq jobs doesnt fail and testing can be done locally

Me-Priyank commented:
UI completely redesigned since
